File: sorts_extension/sorts/exceptions.py
class CredentialsError(Exception):
    def __init__(self) -> None:
        msg = (
            "There has been an error with your Azure credentials (username + token). They might not be correct. "
            "Please refer to https://github.com/fluidattacks/ai-extension-azuredevops#configuring-the-pipeline"
        )
        super(CredentialsError, self).__init__(msg)

File: sorts_extension/exceptions.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.CredentialsError = void 0;
class CredentialsError extends Error {
    constructor() {
        super(`
            You need to set up your Azure credentials (username + token) in your pipeline .yaml.
            Please refer to https://github.com/fluidattacks/ai-extension-azuredevops#configuring-the-pipeline
        `);
        this.name = "CredentialsError";
    }
}
exports.CredentialsError = CredentialsError;
